src/figuresClasses.ts

```diff
@@ -1,11 +1,99 @@
-export interface Figure {}
+type Shape = string | 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Rectangle implements Figure {
+  shape: string = 'rectangle';
```

Comment:
[CODE STYLE] - Enum constant names usually start with capital letter. Since 'shape' is intended to be a constant with a set of predefined values, it should be defined as an enum with capitalized names. Consider refactoring the 'Shape' type into an enum with capitalized constant names, and use it for the 'shape' property in 'Rectangle' and other classes.